Dockerfile

```diff
@@ -168,6 +168,15 @@ RUN if [ "$TARGETPLATFORM" = "linux/arm64" ]; then \
       ; \
     fi
 
+# Install jaxlib on linux/arm64
+# jaxlib, an evofr dependency, does not have official pre-built binaries for
+# linux/arm64. A GitHub user has provided them in a fork repo.
+# https://github.com/google/jax/issues/7097#issuecomment-1110730040
+RUN if [ "$TARGETPLATFORM" = "linux/arm64" ]; then \
+      pip3 install https://github.com/yoziru/jax/releases/download/jaxlib-v0.3.25/jaxlib-0.3.25-cp310-cp310-manylinux2014_aarch64.manylinux_2_17_aarch64.whl \
```

Comment:
_non-blocking_

Don't love using a repackaging by a third-party user… I guess the other option is repackaging it ourselves?

Reply:
Yeah, that could be another adventure so I'll create a separate issue once this PR is merged.

Reply:
Moved to #119.